docs/operator.md

```diff
@@ -154,6 +154,40 @@ spec:
 EOF
 ```
 
+## Deploying the operand with mirror registry configuration
+
+In case user wants to create an installation that uses mirror registry, the following must be executed:
+1. Create and upload a ConfigMap that will include the following keys:
```

Comment:
it seems worth noting that it needs to be in the `assisted-installer` namespace. Trying to imagine the bug being written :sunglasses:

Reply:
the notice regarding namespace is in line 164